apps/src/templates/LoadableComponent.jsx

```diff
@@ -0,0 +1,105 @@
+import React, {useEffect, useState} from 'react';
+import PropTypes from 'prop-types';
+import Spinner from '@cdo/apps/code-studio/pd/components/spinner';
+import i18n from '@cdo/locale';
+import color from '@cdo/apps/util/color';
```

Comment:
Does the new component change the UI at all? If so, mind adding a screenshot to the PR description? (Might be worth it to add a screenshot anyway.)